Container/CSS/avatar.css

```diff
@@ -0,0 +1,135 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: content-box;
+    font-size: 1rem;
+}
+
+.avatar-round {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+
+.avatar-rounded_xS {
+    height: 3rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_S {
+    height: 3.5rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_M {
+    height: 4rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_L {
+    height: 4.5rem;
+    border-radius: 50%;
+}
+
+.avatar-square {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+
+.avatar-square_xS {
+    height: 3rem;
+    border-radius: 25%;
+}
+
+.avatar-square_S {
+    height: 3.5rem;
+    border-radius: 25%;
+}
+
+.avatar-square_M {
+    height: 4rem;
+    border-radius: 25%;
+}
+
+.avatar-square_L {
+    height: 4.5rem;
+    border-radius: 25%;
+}
+
+.avatar-badge-sqaure {
+    display: flex;
+    gap: 1rem;
+}
+
+.avatar-badge_xS,
+.avatar-badge_S,
+.avatar-badge_L,
+.avatar-badge_M {
+    position: relative;
+    display: inline-block;
```

Comment:
Please replace it with a common class name instead of joining multiple classes

Reply:
resolved